popup/folderlist.js

```diff
@@ -291,7 +296,7 @@ class TBFolderList extends HTMLElement {
     }
 
     this.selected = item;
-    this.shadowRoot.querySelector(".folder-list-body").focus();
+    //this.shadowRoot.querySelector(".folder-list-body").focus();
```

Comment:
Same

Reply:
This is the PR for the no focus. I will remove it from here and put it in its own PR.